llm/default_plugins/openai_models.py

```diff
@@ -535,8 +536,16 @@ def get_client(self, key, *, async_=False):
             kwargs["api_key"] = "DUMMY_KEY"
         if self.headers:
             kwargs["default_headers"] = self.headers
+
+        # Configure SSL certificate handling from environment variables
+        ssl_client = _configure_ssl_client(self.model_id)
+
+        if ssl_client:
+            kwargs["http_client"] = ssl_client
+
         if os.environ.get("LLM_OPENAI_SHOW_RESPONSES"):
-            kwargs["http_client"] = logging_client()
+            if "http_client" not in kwargs:
+                kwargs["http_client"] = logging_client()
```

Comment:
Doesn't this mean that `LLM_OPENAI_SHOW_RESPONSES` won't work if a custom SSL configuration is in use?